### ISSUE TITLE
Source code missing for test/data/helloworld binary

### ISSUE THREAD

**Issue by jas4711** [in-toto/in-toto-golang#297]
Hi.  I am packaging in-toto-golang for Debian.

We noticed that the `test/data/helloworld` is a binary executable that is stored in the git repository, and is used during self-tests.  Where is the source code to generate that binary?  Debian does not want to ship binaries that cannot be rebuilt from source code.

I tried replacing it with another binary that I built myself, but I see hard-coded SHA checksums in `in_toto/runlib_test` of the `helloworld` binary, and I'm not sure if there are other properties this binary must have in order for the self-tests to work.  Can you help me?  I will disable the self-tests that depend in this binary in order to proceed with inclusion into Debian.

Thanks,
Simon

**Comment by jas4711** [comment]
For reference, here is the patch we use to disable the `helloworld` self-test:

https://salsa.debian.org/go-team/packages/in-toto-golang/-/blob/1c7eac08f9cc1659e49fc74c57f7959a11133cf0/debian/patches/0001-Disable-test-that-require-sourceless-test-data-hello.patch

/Simon

**Comment by adityasaky** [comment]
IIRC it's only the hash but I can take another look. FWIW, we're evaluating the future of in-toto-golang and how it may be merged with github.com/in-toto/witness / go-witness, so perhaps it's best to hit pause on packaging this one? Is this for a specific Debian-based use case?

**Comment by adityasaky** [comment]
No, the consolidation effort will likely take longer than that.
